nimare/annotate/lda.py

```diff
@@ -92,6 +92,14 @@ def __init__(
         if not op.isdir(text_dir):
             LGR.info("Texts folder not found. Creating text files...")
             os.mkdir(text_dir)
+            # Remove rows with empty text cells
+            #text_df.dropna(subset=[text_column],inplace=True)
```

Comment:
Does this not work?

Reply:
Yes, it works:
```
orig_ids = text_df["id"].tolist()
text_df.dropna(subset=[text_column],inplace=True)
keep_ids = text_df["id"].tolist()
```

Reply:
I'd go with that then, although I'd prefer to operate on a copy instead (i.e., drop `inplace=True` and assign to a fresh `text_df`).